Update directory-country.md (#8301)
magento/devdocs: src/guides/v2.3/graphql/queries/directory-country.md

  - /guides/v2.3/graphql/reference/directory.html
---

> Use the `country` query to retrieve information about a specific country. Dont confuse with [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve list of all the available country in the system.

Reviewer: ```suggestion
Use the `country` query to retrieve information about a specific country. Do not confuse with [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve list of all the available countries in the system.
```

> Use the `country` query to retrieve information about a specific country. Do not confuse with [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve list of all the available countries in the system.

Reviewer: ```suggestion
Use the `country` query to retrieve information about a specific country. Do not confuse with the [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve a list of all the available countries in the system.
```

> Use the `country` query to retrieve information about a specific country. Do not confuse with the [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve a list of all the available countries in the system.

Reviewer 2: ```suggestion
Use the `country` query to retrieve information about a specific country. The [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query is used to retrieve a list of countries available in the system.
```

## Syntax

`{country(id: String) {Country}}`

## Example usage

> The following query returns information about Australia. You must have to pass `id` as two-letter abbreviation of the country to retrieve country detalis.

Reviewer: ```suggestion
The following query returns information about Australia. You must pass `id` as a two-letter abbreviation of the country to retrieve country details.
```

> The following query returns information about Australia. You must pass `id` as a two-letter abbreviation of the country to retrieve country details.

Reviewer 2: ```suggestion
The following query uses a two-letter abbreviation for the country ID (id: "AU"), which returns information about Australia.
```
